ERROR unable to locate installed forge server jar - FTB Cotton modpack

My cmd: `docker run -d -v /mc-data:/data     -e TYPE=FTBA -e FTB_MODPACK_ID=86    -p 25565:25565 -e EULA=TRUE -e SERVER_NAME=Whitelisted -e DIFFICULTY=normal -e INIT_MEMORY=4G -e MAX_MEMORY=6G --restart unless-stopped --name whitelisted itzg/minecraft-server:java8-multiarch`

Snippet of the logs:
`2021/06/07 20:26:01 Downloaded minecraft_server.1.16.5.jar from https://launcher.mojang.com/v1/objects/1b557e7b033b583cd9f66746b7a9ab1ec1673ced/server.jar
2021/06/07 20:26:01 Downloaded 228 successful, 0 failed, 0 incomplete.
2021/06/07 20:26:01 Installing Fabric
2021/06/07 20:26:01 Getting downloads for Fabric
2021/06/07 20:26:03 Installed!
[init] ERROR unable to locate the installed forge server jar
fabric-1.16.5-0.11.3-server-launch.jar
minecraft_server.1.16.5.jar
`

I'm pretty sure my case is specific to the FTB Cotton modpack since it downloads a fabric jar as opposed to a forge jar?
For comparison's sake I replaced FTB_MODPACK_ID=86 (which is FTB Cotton) with a different FTB modpack (79, which is Direwolf20 1.16), and the server started up perfectly fine).

It looks from the repo that there is a 'Fabric' type - I tried setting `TYPE=Fabric` and got a different error:

```
server_1  | [10:01:16] [main/INFO]: Fabric is preparing JARs on first launch, this may take a few seconds...
server_1  | Mapping source name conflicts detected:
server_1  |   ajl METHOD a ((Lcom/mojang/datafixers/Typed;)Lcom/mojang/datafixers/Typed;) -> [ahw/method_24242, ako/method_5003, ajl/method_26302, alg/method_5093, ait/method_22375, ain/method_15592]
server_1  |   aky METHOD a ((Lcom/mojang/datafixers/Typed;)Lcom/mojang/datafixers/Typed;) -> [aky/method_26302, ahw/method_24242, ako/method_5003, alg/method_5093, ait/method_22375, ain/method_15592]
server_1  |   alk METHOD a ((Lcom/mojang/datafixers/Typed;)Lcom/mojang/datafixers/Typed;) -> [alk/method_26302, ahw/method_24242, ako/method_5003, alg/method_5093, ait/method_22375, ain/method_15592]
server_1  |   bgh METHOD j (()V) -> [bgm/method_6858, bgh/method_6860, bgf/method_6844]
server_1  |   bab METHOD j (()V) -> [bgm/method_6858, bgf/method_6844, bab/method_6860]
server_1  |   bdz METHOD j (()V) -> [bdz/method_6860, bgm/method_6858, bgf/method_6844]
server_1  |   bhs METHOD j (()V) -> [bgm/method_6858, bhs/method_6860, bgf/method_6844]
server_1  |   bat METHOD j (()V) -> [bat/method_6860, bgm/method_6858, bgf/method_6844]
server_1  |   ako METHOD a ((Lcom/mojang/datafixers/Typed;)Lcom/mojang/datafixers/Typed;) -> [ako/method_26302, ahw/method_24242, ako/method_5003, alg/method_5093, ait/method_22375, ain/method_15592]
server_1  |   bgo METHOD j (()V) -> [bgm/method_6858, bgf/method_6844, bgo/method_6860]
server_1  |   bds METHOD j (()V) -> [bds/method_6860, bgm/method_6858, bgf/method_6844]
server_1  |   bgr METHOD j (()V) -> [bgm/method_6858, bgr/method_6860, bgf/method_6844]
server_1  |   bdh METHOD j (()V) -> [bgm/method_6858, bgf/method_6844, bdh/method_6860]
server_1  |   bgb METHOD j (()V) -> [bgm/method_6858, bgb/method_6860, bgf/method_6844]
server_1  |   azy METHOD j (()V) -> [bgm/method_6858, azy/method_6860, bgf/method_6844]
server_1  |   bfw METHOD j (()V) -> [bgm/method_6858, bfw/method_6860, bgf/method_6844]
server_1  |   bea METHOD j (()V) -> [bgm/method_6858, bgf/method_6844, bea/method_6860]
server_1  |   bal METHOD j (()V) -> [bal/method_6860, bgm/method_6858, bgf/method_6844]
server_1  |   bfj METHOD j (()V) -> [bgm/method_6858, bfj/method_6860, bgf/method_6844]
server_1  |   bco METHOD j (()V) -> [bgm/method_6858, bco/method_6860, bgf/method_6844]
server_1  |   ago METHOD a ((Lcom/mojang/datafixers/Typed;)Lcom/mojang/datafixers/Typed;) -> [ago/method_26302, ahw/method_24242, ako/method_5003, alg/method_5093, ait/method_22375, ain/method_15592]
server_1  |   baa METHOD j (()V) -> [baa/method_6860, bgm/method_6858, bgf/method_6844]
server_1  |   agu METHOD a ((Lcom/mojang/datafixers/Typed;)Lcom/mojang/datafixers/Typed;) -> [agu/method_26302, ahw/method_24242, ako/method_5003, alg/method_5093, ait/method_22375, ain/method_15592]
server_1  |   ahw METHOD a ((Lcom/mojang/datafixers/Typed;)Lcom/mojang/datafixers/Typed;) -> [ahw/method_24242, ako/method_5003, alg/method_5093, ahw/method_26302, ait/method_22375, ain/method_15592]
server_1  |   bgv METHOD j (()V) -> [bgv/method_6860, bgm/method_6858, bgf/method_6844]
server_1  |   bah METHOD j (()V) -> [bah/method_6860, bgm/method_6858, bgf/method_6844]
server_1  |   bao METHOD j (()V) -> [bgm/method_6858, bao/method_6860, bgf/method_6844]
server_1  |   bek METHOD j (()V) -> [bek/method_6860, bgm/method_6858, bgf/method_6844]
server_1  |   alg METHOD a ((Lcom/mojang/datafixers/Typed;)Lcom/mojang/datafixers/Typed;) -> [ahw/method_24242, ako/method_5003, alg/method_5093, alg/method_26302, ait/method_22375, ain/method_15592]
server_1  |   aih METHOD a ((Lcom/mojang/datafixers/Typed;)Lcom/mojang/datafixers/Typed;) -> [ahw/method_24242, ako/method_5003, aih/method_26302, alg/method_5093, ait/method_22375, ain/method_15592]
server_1  |   agr METHOD a ((Lcom/mojang/datafixers/Typed;)Lcom/mojang/datafixers/Typed;) -> [ahw/method_24242, ako/method_5003, alg/method_5093, agr/method_26302, ait/method_22375, ain/method_15592]
server_1  |   bap METHOD j (()V) -> [bap/method_6860, bgm/method_6858, bgf/method_6844]
server_1  |   aah METHOD j (()V) -> [aah/method_6860, bgm/method_6858, bgf/method_6844]
server_1  |   bcu METHOD j (()V) -> [bgm/method_6858, bcu/method_6860, bgf/method_6844]
server_1  |   aqg METHOD j (()V) -> [aqg/method_6860, bgm/method_6858, bgf/method_6844]
server_1  |   bcw METHOD j (()V) -> [bgm/method_6858, bcw/method_6860, bgf/method_6844]
server_1  |   bgf METHOD j (()V) -> [bgf/method_6860, bgm/method_6858, bgf/method_6844]
server_1  |   bbq METHOD j (()V) -> [bbq/method_6860, bgm/method_6858, bgf/method_6844]
server_1  |   bga METHOD j (()V) -> [bga/method_6860, bgm/method_6858, bgf/method_6844]
server_1  |   aqn METHOD j (()V) -> [bgm/method_6858, bgf/method_6844, aqn/method_6860]
server_1  |   azu METHOD j (()V) -> [azu/method_6860, bgm/method_6858, bgf/method_6844]
server_1  |   aip METHOD a ((Lcom/mojang/datafixers/Typed;)Lcom/mojang/datafixers/Typed;) -> [ahw/method_24242, ako/method_5003, alg/method_5093, aip/method_26302, ait/method_22375, ain/method_15592]
server_1  |   bdw METHOD j (()V) -> [bgm/method_6858, bdw/method_6860, bgf/method_6844]
server_1  |   agb$1 METHOD a ((Lcom/mojang/datafixers/Typed;)Lcom/mojang/datafixers/Typed;) -> [agb$1/method_26302, ahw/method_24242, ako/method_5003, alg/method_5093, ait/method_22375, ain/method_15592]
server_1  |   bgi METHOD j (()V) -> [bgi/method_6860, bgm/method_6858, bgf/method_6844]
server_1  |   bhu METHOD j (()V) -> [bhu/method_6860, bgm/method_6858, bgf/method_6844]
server_1  |   aht METHOD a ((Lcom/mojang/datafixers/Typed;)Lcom/mojang/datafixers/Typed;) -> [ahw/method_24242, ako/method_5003, aht/method_26302, alg/method_5093, ait/method_22375, ain/method_15592]
server_1  |   agt METHOD a ((Lcom/mojang/datafixers/Typed;)Lcom/mojang/datafixers/Typed;) -> [agt/method_26302, ahw/method_24242, ako/method_5003, alg/method_5093, ait/method_22375, ain/method_15592]
server_1  |   bhl METHOD j (()V) -> [bgm/method_6858, bgf/method_6844, bhl/method_6860]
server_1  |   ahj METHOD a ((Lcom/mojang/datafixers/Typed;)Lcom/mojang/datafixers/Typed;) -> [ahw/method_24242, ako/method_5003, ahj/method_26302, alg/method_5093, ait/method_22375, ain/method_15592]
server_1  |   beb METHOD j (()V) -> [bgm/method_6858, bgf/method_6844, beb/method_6860]
server_1  |   bhn METHOD j (()V) -> [bhn/method_6860, bgm/method_6858, bgf/method_6844]
server_1  |   apz METHOD j (()V) -> [bgm/method_6858, apz/method_6860, bgf/method_6844]
server_1  |   bgy METHOD j (()V) -> [bgy/method_6860, bgm/method_6858, bgf/method_6844]
server_1  |   bej METHOD j (()V) -> [bgm/method_6858, bgf/method_6844, bej/method_6860]
server_1  |   aql METHOD j (()V) -> [aql/method_6860, bgm/method_6858, bgf/method_6844]
server_1  |   bgt METHOD j (()V) -> [bgt/method_6860, bgm/method_6858, bgf/method_6844]
server_1  |   ags METHOD a ((Lcom/mojang/datafixers/Typed;)Lcom/mojang/datafixers/Typed;) -> [ags/method_26302, ahw/method_24242, ako/method_5003, alg/method_5093, ait/method_22375, ain/method_15592]
server_1  |   bbb METHOD j (()V) -> [bbb/method_6860, bgm/method_6858, bgf/method_6844]
server_1  |   bgl METHOD j (()V) -> [bgl/method_6860, bgm/method_6858, bgf/method_6844]
server_1  |   aqa METHOD j (()V) -> [aqa/method_6860, bgm/method_6858, bgf/method_6844]
server_1  |   ajs METHOD a ((Lcom/mojang/datafixers/Typed;)Lcom/mojang/datafixers/Typed;) -> [ajs/method_26302, ahw/method_24242, ako/method_5003, alg/method_5093, ait/method_22375, ain/method_15592]
server_1  |   agb$2 METHOD a ((Lcom/mojang/datafixers/Typed;)Lcom/mojang/datafixers/Typed;) -> [agb$2/method_26302, ahw/method_24242, ako/method_5003, alg/method_5093, ait/method_22375, ain/method_15592]
server_1  |   ahl METHOD a ((Lcom/mojang/datafixers/Typed;)Lcom/mojang/datafixers/Typed;) -> [ahw/method_24242, ako/method_5003, alg/method_5093, ahl/method_26302, ait/method_22375, ain/method_15592]
server_1  |   bhv METHOD j (()V) -> [bhv/method_6860, bgm/method_6858, bgf/method_6844]
server_1  |   ale METHOD a ((Lcom/mojang/datafixers/Typed;)Lcom/mojang/datafixers/Typed;) -> [ale/method_26302, ahw/method_24242, ako/method_5003, alg/method_5093, ait/method_22375, ain/method_15592]
server_1  |   ahz METHOD a ((Lcom/mojang/datafixers/Typed;)Lcom/mojang/datafixers/Typed;) -> [ahw/method_24242, ako/method_5003, alg/method_5093, ahz/method_26302, ait/method_22375, ain/method_15592]
server_1  |   bee METHOD j (()V) -> [bgm/method_6858, bgf/method_6844, bee/method_6860]
server_1  |   bgc METHOD j (()V) -> [bgm/method_6858, bgf/method_6844, bgc/method_6860]
server_1  |   bht METHOD j (()V) -> [bgm/method_6858, bgf/method_6844, bht/method_6860]
server_1  |   aig METHOD a ((Lcom/mojang/datafixers/Typed;)Lcom/mojang/datafixers/Typed;) -> [ahw/method_24242, ako/method_5003, alg/method_5093, aig/method_26302, ait/method_22375, ain/method_15592]
server_1  |   bcv METHOD j (()V) -> [bgm/method_6858, bcv/method_6860, bgf/method_6844]
server_1  |   baz METHOD j (()V) -> [bgm/method_6858, bgf/method_6844, baz/method_6860]
server_1  |   ain METHOD a ((Lcom/mojang/datafixers/Typed;)Lcom/mojang/datafixers/Typed;) -> [ain/method_26302, ahw/method_24242, ako/method_5003, alg/method_5093, ait/method_22375, ain/method_15592]
server_1  |   ald METHOD a ((Lcom/mojang/datafixers/Typed;)Lcom/mojang/datafixers/Typed;) -> [ahw/method_24242, ako/method_5003, ald/method_26302, alg/method_5093, ait/method_22375, ain/method_15592]
server_1  |   baf METHOD j (()V) -> [bgm/method_6858, bgf/method_6844, baf/method_6860]
server_1  |   ajv METHOD a ((Lcom/mojang/datafixers/Typed;)Lcom/mojang/datafixers/Typed;) -> [ahw/method_24242, ako/method_5003, ajv/method_26302, alg/method_5093, ait/method_22375, ain/method_15592]
server_1  |   bgm METHOD j (()V) -> [bgm/method_6860, bgm/method_6858, bgf/method_6844]
server_1  |   bdc METHOD j (()V) -> [bdc/method_6860, bgm/method_6858, bgf/method_6844]
server_1  |   bdx METHOD j (()V) -> [bgm/method_6858, bgf/method_6844, bdx/method_6860]
server_1  |   ahb METHOD a ((Lcom/mojang/datafixers/Typed;)Lcom/mojang/datafixers/Typed;) -> [ahb/method_26302, ahw/method_24242, ako/method_5003, alg/method_5093, ait/method_22375, ain/method_15592]
server_1  |   aqm METHOD j (()V) -> [bgm/method_6858, aqm/method_6860, bgf/method_6844]
server_1  |   bge METHOD j (()V) -> [bge/method_6860, bgm/method_6858, bgf/method_6844]
server_1  |   bed METHOD j (()V) -> [bed/method_6860, bgm/method_6858, bgf/method_6844]
server_1  |   agn METHOD a ((Lcom/mojang/datafixers/Typed;)Lcom/mojang/datafixers/Typed;) -> [ahw/method_24242, ako/method_5003, alg/method_5093, agn/method_26302, ait/method_22375, ain/method_15592]
server_1  |   bcn METHOD j (()V) -> [bgm/method_6858, bcn/method_6860, bgf/method_6844]
server_1  |   ajz METHOD a ((Lcom/mojang/datafixers/Typed;)Lcom/mojang/datafixers/Typed;) -> [ahw/method_24242, ako/method_5003, ajz/method_26302, alg/method_5093, ait/method_22375, ain/method_15592]
server_1  |   ait METHOD a ((Lcom/mojang/datafixers/Typed;)Lcom/mojang/datafixers/Typed;) -> [ahw/method_24242, ako/method_5003, alg/method_5093, ait/method_26302, ait/method_22375, ain/method_15592]
server_1  | Exception in thread "main" java.lang.RuntimeException: Failed to setup Fabric server environment!
server_1  |     at net.fabricmc.loader.launch.server.FabricServerLauncher.main(FabricServerLauncher.java:58)
server_1  | Caused by: java.lang.RuntimeException: java.lang.RuntimeException: An exception occurred when launching the server!
server_1  |     at net.fabricmc.loader.launch.server.FabricServerLauncher.setup(FabricServerLauncher.java:116)
server_1  |     at net.fabricmc.loader.launch.server.FabricServerLauncher.main(FabricServerLauncher.java:56)
server_1  | Caused by: java.lang.RuntimeException: An exception occurred when launching the server!
server_1  |     at net.fabricmc.loader.launch.server.FabricServerLauncher.launch(FabricServerLauncher.java:70)
server_1  |     at net.fabricmc.loader.launch.server.FabricServerLauncher.setup(FabricServerLauncher.java:114)
server_1  |     ... 1 more
server_1  | Caused by: java.lang.reflect.InvocationTargetException
server_1  |     at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
server_1  |     at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
server_1  |     at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
server_1  |     at java.lang.reflect.Method.invoke(Method.java:498)
server_1  |     at net.fabricmc.loader.launch.server.FabricServerLauncher.launch(FabricServerLauncher.java:68)
server_1  |     ... 2 more
server_1  | Caused by: java.lang.RuntimeException: Unfixable conflicts
server_1  |     at net.fabricmc.tinyremapper.TinyRemapper.handleConflicts(TinyRemapper.java:773)
server_1  |     at net.fabricmc.tinyremapper.TinyRemapper.propagate(TinyRemapper.java:673)
server_1  |     at net.fabricmc.tinyremapper.TinyRemapper.mrjRefresh(TinyRemapper.java:948)
server_1  |     at net.fabricmc.tinyremapper.TinyRemapper.apply(TinyRemapper.java:804)
server_1  |     at net.fabricmc.tinyremapper.TinyRemapper.apply(TinyRemapper.java:778)
server_1  |     at net.fabricmc.loader.launch.common.FabricLauncherBase.deobfuscate0(FabricLauncherBase.java:185)
server_1  |     at net.fabricmc.loader.launch.common.FabricLauncherBase.deobfuscate(FabricLauncherBase.java:118)
server_1  |     at net.fabricmc.loader.launch.knot.Knot.init(Knot.java:117)
server_1  |     at net.fabricmc.loader.launch.knot.KnotServer.main(KnotServer.java:28)
server_1  |     ... 7 more
server_1  | 2021-06-19T10:01:17.848Z    WARN    mc-server-runner        sub-process failed      {"exitCode": 1}
server_1  | 2021-06-19T10:01:17.848Z    INFO    mc-server-runner        Done
minecraftftbcotton_server_1 exited with code 1
```

--- a/start-deployFTBA
+++ b/start-deployFTBA
@@ -52,12 +52,14 @@ fi
 
 isDebugging && cat version.json
 forgeVersion=$(jq -r '.targets|unique[] | select(.name == "forge") | .version' version.json)
+fabricVersion=$(jq -r '.targets|unique[] | select(.name == "fabric") | .version' version.json)
 mcVersion=$(jq -r '.targets|unique[] | select(.name == "minecraft") | .version' version.json)
 
 variants=(
   forge-${mcVersion}-${forgeVersion}.jar
   forge-${mcVersion}-${forgeVersion}-universal.jar
   forge-${mcVersion}-${forgeVersion}-${mcVersion}-universal.jar
+  fabric-${mcVersion}-${fabricVersion}-server-launch.jar
 )
 for f in ${variants[@]}; do
   if [ -f $f ]; then
@@ -66,7 +68,7 @@ for f in ${variants[@]}; do
   fi
 done
 if ! [ -v SERVER ]; then
-  log "ERROR unable to locate the installed forge server jar"
+  log "ERROR unable to locate the installed FTB server jar"
   ls *.jar
   exit 2
 fi

